### DISCUSSION TITLE
Olá desenvolvedores e entusiastas de código 🤩

### DISCUSSION THREAD

**Discussion by AbraaoAlves** [dcTeam23/course#1] [General]
👋🏽 Olá futuros colegas desenvolvedores!

Estamos utilizando as Discussões como um lugar para nos conectar com membros da nossa organização. Esperamos que você:

Faça perguntas que você está se perguntando!
E compartilhe suas ideias!

Aproveite esse espaço para interagir com os colegas, dê as boas-vindas aos outros, [seja respeitoso, empático e tenha uma mente aberta](https://docs.github.com/pt/site-policy/github-terms/github-community-guidelines). 
Lembre-se de que esta é uma comunidade só nossa (apenas os membros da organização podem ver essas discussões) e que construímos juntos 💪🏽.

Para começar, comente abaixo com uma introdução sobre você e nos conte sobre sobre suas expectativas com essa comunidade.

**Comment by jmtannus** [comment]
Olá! 👋Me chamo Janaína. 
Sou uma pessoa aprendendo a ver o mundo com uma nova perspectiva. Amo esportes, viajar, bater papo, cantar, meditar, cozinhar e tomar☕ com os colegas da dcTeam23. 

A cada dia me encanto com o que sou capaz de aprender, de ensinar e com esse mundo desafiador que é a TI.🤓

Minha expectativa é de compartilhar experiências e ver a evolução de todos, mostrar pro mundo e para nós mesmos que somos/seremos bons fullstacks e que no final da jornada todos os esforços valerão a pena! 🤜🤛
